### PR TITLE
Update and Re-organize Readme

### PR DESCRIPTION
**Pull request by danaelhe**
- Moved the contribution section to it's own file
- Made the sections more prominent
- Ensured content flowed better

### CODE REVIEW

**Review by ChiefMateStarbuck — DISMISSED** [review]

File: .markdownlint.yml [hunk 1]
```diff
@@ -6,5 +6,7 @@ MD033: false
 
 MD024: false
 
+MD001: false
```
Comment (ChiefMateStarbuck) [inline .markdownlint.yml:9]: what does this do?

**Review by ChiefMateStarbuck — APPROVED** [review]
🖌️

File: .markdownlint.yml (hunk 1 above)
Comment (danaelhe) [reply .markdownlint.yml:9]: It suppresses some of  [markdownlint-cli](https://github.com/igorshubovych/markdownlint-cli)'s rules. This one in particular suppresses the header increments:
`MD001/heading-increment/header-increment Heading levels should only increment by one level at a time [Expected: h2; Actual: h4]`

I think it's easier to read when the headers are more prominent than it's subheaders

File: README.md [hunk 2]
```diff
@@ -35,11 +35,22 @@ To install from source:
...
+> A quick guide to getting started with the client.
 
-Find below a working example for GET a ssh_key ([per this http request](https://docs.digitalocean.com/reference/api/api-reference/#operation/sshKeys_list)) and printing the ID associated with the ssh key. If you'd like to try out this quick example, you can follow [these instructions](https://docs.digitalocean.com/products/droplets/how-to/add-ssh-keys/) to add ssh keys to your DO account.
+`pydo` must be initialized with `pydo.Client()`. A DigitalOcean API Token is required. The token can be passed explicitly to `pydo.Client()`, as such:
+
+```python
+import os
+from pydo import Client
+
+client = Client(token=os.getenv("$DIGITALOCEAN_TOKEN"))
```
Comment (merwok) [inline README.md:48]: `getenv` takes the environment variable name as parameter, without `$` from shell syntax

**Pull request merged**